crmprtd/infill.py

```diff
@@ -54,9 +54,9 @@ def download_and_process(
     network_name: str,
     log_args: List[str],
     download_args: List[str],
-    cache_filename: str = None,
-    connection_string: str = None,
-    dry_run: bool = None,
+    cache_filename: Optional[str] = None,
+    connection_string: Optional[str] = None,
+    dry_run: bool = True,
```

Comment:
How about default `False`? There's no "not dry-run" except for `--dry_run=False`.